packages/core/__tests__/language-server/references-ts-plugin.test.ts

```diff
@@ -0,0 +1,230 @@
+import {
+  getSharedTestWorkspaceHelper,
+  teardownSharedTestWorkspaceAfterEach,
+  prepareDocument,
+  testWorkspacePath,
+  extractCursor,
+  extractCursors,
+} from 'glint-monorepo-test-utils';
+import { describe, afterEach, test, expect } from 'vitest';
+import { stripIndent } from 'common-tags';
+import { URI } from 'vscode-uri';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+
+describe('Language Server: References (ts plugin)', () => {
+  afterEach(teardownSharedTestWorkspaceAfterEach);
+
+  test.only('component references', async () => {
+    const [offset, content] = extractCursor(stripIndent`
+      import Component from '@glimmer/component';
+
+      export default class Gre%eting extends Component {
+        private nested = Math.random() > 0.5;
+
+        <template>
+          {{#if this.nested}}
+            <Greeting />!
+          {{else}}
+            Hello!
+          {{/if}}
+        </template>
+      }
+    `);
+
+    const greetingDoc = await prepareDocument(
+      'ts-template-imports-app/src/ephemeral-greeting.gts',
+      'glimmer-ts',
+      content,
+    )
+
+    await prepareDocument(
+      'ts-template-imports-app/src/ephemeral-app.gts',
+      'glimmer-ts',
+      stripIndent`
+        import Component from '@glimmer/component';
+        import Greeting from './ephemeral-greeting.gts';
+
+        export default class Application extends Component {
+          <template>
+            <Greeting />
+          </template>
+        }
+      `,
+    );
+
+    expect(
+      await performReferencesRequest(
+        greetingDoc,
+        offset,
+      ),
+    ).toMatchInlineSnapshot(`
```

Comment:
This is wrong; there should be 4 references total, but 2 references are missing from `ephemeral-app.gts`. I've tested in F5 debug mode and it seems to be working; wondering if there is something strange about the test setup.

Reply:
Fixed via on-disk empty-fixture.gts